stroeer/page/section/v1/section_page.proto

```diff
@@ -59,51 +60,49 @@ message StageConfiguration {
   repeated stroeer.core.v1.Reference references = 2;
 }
 
-// A single generic item of a stage.
+// A generic model to represent what can be part of a stage.
 message StageItem {
-  // Width instructions for rendering the single item.
-  StageItemWidth stage_item_width = 1;
-  // At most one field will be set at the same time.
-  // The delegate of the StageItem can be another Stage, an editorial article or a Commercial.
+  // Optional flags/hints for customization independent of the stage_item_type
+  map<string, string> fields = 1;
```

Comment:
maybe reserve some space here because its expected that the `oneof` will grow in the future